include/spatio_temporal_voxel_layer/measurement_buffer.hpp

```diff
@@ -126,6 +130,7 @@ class MeasurementBuffer
   double _min_z, _max_z, _vertical_fov, _horizontal_fov, _decay_acceleration;
   double _voxel_size;
   bool _marking, _clearing, _voxel_filter, _clear_buffer_after_reading;
+  geometry_msgs::TransformStamped tf_stamped;
```

Comment:
This doesnt need to be a class variable, you can just instantiate it in the use below as 
```
geometry_msgs::TransformStamped tf_stamped = _buffer.lookupTransform(_global_frame, cloud.header.frame_id.c_str(), ros::Time(0));

```